src/main/java/pages/CartPage.java

```diff
@@ -3,29 +3,35 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 
-public class CartPage extends BasePage{
-    private By items= By.xpath("//tbody//tr");
-    private By itemsQuantity=By.xpath("//tbody//button");
-    private By itemsPrice=By.xpath("//tbody//td[@class=\"cart_price\"]//p");
-    private By itemsTotalPrice=By.xpath("//tbody//td[@class=\"cart_total\"]//p");
+public class CartPage extends BasePage {
+    private By items = By.xpath("//tbody//tr");
```

Comment:
Not the best locators, from line 6 to line 10
You should link this to id attribute